### PR TITLE
Task4 - Sub task 1 and 2 completed

### PR DESCRIPTION
**Pull request by asintdivya123**

### CODE REVIEW

File: assignment4/src/main/java/com/asint/task4/cache/Page.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	 * manually download the requested url contents and
+	 * update the page in the cache.
+	 * 
+	 * @param url
+	 * @param refresh
+	 * @throws IOException
+	 */
+	public Page(URL url, boolean refresh) throws IOException {
+		super();
+		// TODO: Implement
```
Comment (StillSomehowSane) [inline assignment4/src/main/java/com/asint/task4/cache/Page.java]: Remove

File: assignment4/src/main/java/com/asint/task4/controller/CacheController.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	
+	@GetMapping("/scache/api/admin/all")
+	public ResponseEntity<?> getAllPagesInCache() {
+		// TODO: Implement
+		return null;
+	}
+
+	@GetMapping("/scache/api/refresh")
+	public ResponseEntity<?> refreshCache(@RequestParam(value = "url", required=false, defaultValue = "") String url) {
+		// TODO: Implement
```
Comment (StillSomehowSane) [inline assignment4/src/main/java/com/asint/task4/controller/CacheController.java]: Remove

File: assignment4/src/main/java/com/asint/task4/controller/CacheController.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+		}
+	}
+	
+	@GetMapping("/scache/api/admin/all")
+	public ResponseEntity<?> getAllPagesInCache() {
+		// TODO: Implement
+		return null;
+	}
+
+	@GetMapping("/scache/api/refresh")
```
Comment (StillSomehowSane) [inline assignment4/src/main/java/com/asint/task4/controller/CacheController.java]: add v1

**Review by StillSomehowSane — CHANGES_REQUESTED** [review]

**Comment by asintdivya123** [comment]
requested changes are made

**Review by StillSomehowSane — APPROVED** [review]
Looks good

**Comment by asintdivya123** [comment]
Okay 👍

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**